README.md

```diff
@@ -1,3 +1,7 @@
+# This Gem has been Deprecated
+
+If you are looking to build rails apps with Gnarly Opinions, check out the `init` command of [`gnar-cli`].
```

Comment:
should `gnar-cli` link to the Gnar CLI repo or somewhere similar?

Reply:
Duh, Yep!